src/alpha-lab/scanpath-rim.md

```diff
@@ -13,123 +13,85 @@ tags: [Pupil Invisible, Neon, Cloud]
-IDs are displayed in black text with a white border to make them stand out from the background. If you adjust the font 
-size, it's also recommended to increase the values of `font_thick_w` and `font_thick_b` to maintain visual contrast.
-``` python
-def draw_on_frame(...):
-# text aesthetics
-    font = cv2.FONT_HERSHEY_DUPLEX
-    font_size = 1
-    font_thick_w = 3 
-    font_thick_b = 1
-...
-```
-### My name is legend
-The script includes two functions for creating a legend to display the wearer names and corresponding colors:
-
-1. `draw_name_legend()`: This function creates a legend box that displays only the name of the wearer on their individual scanpath video and image.
-2. `draw_all_names_legend()`: This function creates a legend that displays all the wearer names on the final general scanpath image.
-
-To customize the appearance of the legend, such as the position, dimensions, or colors of the rectangular white box or the colored line, 
-you can modify the following parameters in both functions:
-
-- `r_end_point` - x and y values of the ending coordinates of the rectangular legend box
-- `r_start_point` - x and y values of the starting coordinates of the rectangular legend box
-- `l_end_point` - x and y values of the ending coordinates of the colored line
-- `l_start_point` - x and y values of the starting coordinates of the colored line
-- In `cv2.rectangle`, edit `color` to set a new color for the legend box
-- In `cv2.line`, edit `thickness` to set a new width for the colored line
+## Extending current tools
+The [Reference Image Mapper](/enrichments/reference-image-mapper) enrichment available in Pupil Cloud is a tool that maps gaze onto
+2D images and can subsequently generate heatmaps. However, it currently does not support the production of scanpath visualizations.
+Thus, we chose to develop a script that shows you how to build your own scanpaths using Reference Image Mapped data.
+
+
+## Steps
+1. Run a [Reference Image Mapper enrichment](https://docs.pupil-labs.com/enrichments/reference-image-mapper/) and download the results
+2. Download [this script](https://gist.github.com/elepl94/9f669c4d81e455cf2095957831219664) and follow the [installation instructions](https://gist.github.com/elepl94/9f669c4d81e455cf2095957831219664#installation)
+
+## Review the scanpaths
+After the script has completed its execution, you'll find the resulting scanpath visualizations stored in a newly created 
+sub-folder named "scanpath." For each participant, you will obtain a reference image with the scanpath superimposed on it. 
+You will also find a video featuring a dynamic scanpath overlay. Finally, if you had multiple participants, an aggregated 
+visualization combining all participants' scanpaths will be available, enabling a more comprehensive overview of the subjects'
+gaze behavior.
+
+<div style="display: flex; justify-content: space-between;">
```

Comment:
Other images come below. I would just leave them as single images in the column, no need to do side by unless you want the reader to compare the two images.